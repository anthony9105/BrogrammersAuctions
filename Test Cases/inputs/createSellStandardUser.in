create
newUser12345   
sell-standard