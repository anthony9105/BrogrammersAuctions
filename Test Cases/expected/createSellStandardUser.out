Enter new username
Enter type of user (admin, buy-standard, or sell-standard)
Success.  New user added.